src/inmemorystorage.ts

```diff
@@ -82,7 +79,9 @@ export class InMemoryStorage extends CachingLayer {
     return Promise.resolve();
   }
 
+  // NOTE: will be overwritten by eventHandlingMíxin
   _emit(...args): void {
```

Comment:
A function that only throws should have a return type of `never`, which TypeScript actually knows and takes into account when the function is used then:

```suggestion
  _emit(...args): never {
```

However, I'm not sure if it's necessary to have this function included here in the first place.